docker-hub Admin right gives access to build logs

tristan0x opened the pull request.

unclejack commented:
@tristan0x Please squash your commits into one and force push to this branch.

tristan0x commented:
@unclejack it's done!

SvenDowideit commented:
LGTM - @moxiegirl

moxiegirl commented on docs/docker-hub/accounts.md:
```diff
@@ -79,7 +79,7 @@ repository access rights to groups:
   Hub.
 - `Admin` access allows the user to modify the repositories "Description", "Collaborators" rights,
-  "Mark as unlisted", "Public/Private" status and "Delete".
+  "Mark as unlisted", "Public/Private" status and "Delete", and to retrieve build logs.
```
"and to retrieve build logs" did you mean:

"Mark as unlisted", "Public/Private" status, and "Delete" to retrieve build logs.

tristan0x replied:
No.
"Automated build" repositories trigger a new build of the Docker image whenever a change occurs in the tracked repository. Every build is recorded and visible through the *builds history* page for users with Read permission. But it is required to have Admin permission to actually read a build log.

moxiegirl replied:
Thanks for the clarification. This section on rights has some additional structural issues. If you don't mind, let's clarify them all.  If you do mind, happy to carry.

----> https://gist.github.com/moxiegirl/1c34957b9edc0f6d528c

### Repository group permissions

Use organization groups to manage the users that can interact with your repositories.

You must be in an organization's `Owners` group to create a new group, Hub
repository, or automated build. As an `Owner`, you then delegate the following
repository access rights to groups:

| Access Right | Description                                                                                                                                                                                                                                                                |
|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `Read`       | Users with this right can view, search, and pull a private repository.                                                                                                                                                                                                     |
| `Write`      | Users with this right can push to non-automated repositories on the Docker Hub.                                                                                                                                                                                            |
| `Admin`      | Users with this right can modify a repository's "Description", "Collaborators" rights. They can also mark a repository as unlisted, change its  "Public/Private" status and "Delete" the repository. Finally, `Admin` rights are required to read the build log on a repo. |
|              |                                                                                                                                                                                                                                                                            |

Regardless of their actual access rights, users with unverified email addresses
have `Read` access to the repository. Once they have verified their address,
they have their full access rights as granted on the organization.

thaJeztah commented:
@tristan0x do you want do update your PR with the changes that @moxiegirl suggested, or do you want us to carry?

tristan0x commented:
Hi Sebastiaan,
I'll update it today.
Regards

On Fri, Jul 3, 2015, 00:30 Sebastiaan van Stijn <notifications@github.com>
wrote:

> @tristan0x <https://github.com/tristan0x> do you want do update your PR
> with the changes that @moxiegirl <https://github.com/moxiegirl>
> suggested, or do you want us to carry?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/14159#issuecomment-118184744>.
>

thaJeztah commented:
@tristan0x thanks!

GordonTheTurtle commented:
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "patch-1" git@github.com:tristan0x/docker.git somewhere
$ cd somewhere
$ git commit --amend -s --no-edit
$ git push -f
~~~

Ammending updates the existing PR. You **DO NOT** need to open a new one.

Commit:
Rewrite DockerHub group permissions documentation

    On behalf of @moxiegirl
    https://gist.github.com/moxiegirl/1c34957b9edc0f6d528c#file-gistfile1-md

Signed-off-by: Tristan Carel <tristan.carel@gmail.com>

thaJeztah commented:
LGTM, thanks!

moxiegirl commented:
cherry-picked
